### PR TITLE
Feature/packages/takeover  #web 0000

### PR DESCRIPTION
**Pull request by Mar0cas**
- Configuration files
- Story files
- Test files
- Component files
- Srouce files

### CODE REVIEW

File: packages/organisms/takeover/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "build": "tsc",
+    "start": "nodemon --watch src --ext ts,tsx,js,json --exec yarn build",
+    "test": "jest",
+    "clean": "sh ../../../scripts/packages/clean.sh -p takeover"
+  },
+  "peerDependencies": {
+    "react": "^16.13.1"
+  },
+  "dependencies": {
+    "@creat/common": "^1.0.3",
```
Comment (rebecatalkdesk) [inline packages/organisms/takeover/package.json:21]: If you need the new changes to common for the cookies, this version will need to be bumped to 1.0.4 here and in the common package, or else the takeover won't have the new utils

File: packages/organisms/takeover/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "@creat/section": "^1.2.0",
+    "@creat/buttons": "^1.0.3",
+    "@creat/image": "~1.0.3",
```
Comment (rebecatalkdesk) [inline packages/organisms/takeover/package.json:24]: Same thing here if you need the svg update.

**Review by rebecatalkdesk — CHANGES_REQUESTED** [review]
Missing the snapshot

File: packages/organisms/takeover/src/Takeover.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            </div>
+            <div className="outer--cta wrapper__cta">
+              <a href="#close-takeover"
+                target="_self"
+                data-area="cta"
+                className="link link--white -outlined -light link--ui"
+                onClick={ Utils.closeTakeover }
+              >
+                continue to talkdesk.com
+                <i className="icon-ui-arrowlink">
```
Comment (rebecatalkdesk) [inline packages/organisms/takeover/src/Takeover.tsx:69]: Use the icon atom

File: packages/organisms/takeover/src/Takeover.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+          <div className="col-14 d-flex flex-column h-100 align-items-start">
+            <a onClick={ Utils.closeTakeover } href="#close-takeover" className="logo">
+              <div className="header__logo">
+                {!notEmptyOrNull(logo) || logo?.map((logo: any, idx: number) => <Image key={idx} {...logo} /> )}
+              </div>
+            </a>
+            <a onClick={ Utils.closeTakeover } href="#close-takeover" className="button--close"></a>
+            <div className="content-wrapper col-14 col-sm-7 pt-5 my-sm-auto pt-sm-0">
+            {!notEmptyOrNull(content) || <Html content={{text: content as any}} className="takeover-title" />}
+            {!ctas || <Buttons cta={ctas} />}
```
Comment (rebecatalkdesk) [inline packages/organisms/takeover/src/Takeover.tsx:59]: ctas can be an empty array, this only checks if there is the prop right?

Comment (Mar0cas) [reply packages/organisms/takeover/src/Takeover.tsx]: Fixed

File: packages/organisms/takeover/src/Takeover.tsx (hunk 3 above)
Comment (Mar0cas) [reply packages/organisms/takeover/src/Takeover.tsx]: fixed

**Review by rebecatalkdesk — APPROVED** [review]

**Pull request merged**